src/static/js/views/faq.html

```diff
@@ -34,44 +34,21 @@ <h4>Frequently Asked Questions</h4>
       <div class="list-group-item" ng-show="questionState(3)">
         <p>More details of packages can be obtained directly from following links provided by respective distros.</p>
         <ul>
-          <li><a href="https://www.suse.com/releasenotes/x86_64/SUSE-SLES/12/#InfraPackArch.SystemZ" target="blank">SUSE Linux Enterprise Server 12</a></li>
-          <li><a href="https://www.suse.com/releasenotes/s390x/SUSE-SLES/15/" target="blank">SUSE Linux Enterprise Server 15</a></li>
-          <li><a href="https://www.suse.com/releasenotes/s390x/SUSE-SLES/15-SP1/" target="blank">SUSE Linux Enterprise Server 15 SP1</a></li>
-          <li><a href="https://www.suse.com/releasenotes/s390x/SUSE-SLES/15-SP2/" target="blank">SUSE Linux Enterprise Server 15 SP2</a></li>
-          <li><a href="https://packagehub.suse.com/#packages" target="blank">SUSE Package Hub</a></li>
-          <li><a href="https://launchpad.net/ubuntu/xenial/s390x" target="blank">Ubuntu 18.04</a></li>
-          <li><a href="https://launchpad.net/ubuntu/bionic/s390x" target="blank">Ubuntu 19.04</a></li>
-          <li><a href="https://launchpad.net/ubuntu/focal/s390x/" target="blank">Ubuntu 20.04</a></li>
-          <li><a href="https://launchpad.net/ubuntu/groovy/s390x/" target="blank">Ubuntu 20.10</a></li>
-          <li><a href="https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/6.10_release_notes/index" target="blank">RHEL 6.10</a></li>
-          <li><a href="https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/package_manifest/package_lists_base_repository#Base-Server" target="blank">RHEL 7</a></li>
-          <li><a href="https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.0_release_notes/index" target="blank">RHEL 8.0</a></li>
-          <li><a href="https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/package_manifest/index" target="blank">RHEL 8.1</a></li>
-          <li><a href="https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.2_release_notes/distribution-of-content-in-rhel-8#repositories" target="blank">RHEL 8.2</a></li>
-          <li><a href="https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.3_release_notes/distribution-of-content-in-rhel-8#repositories" target="blank">RHEL 8.3</a></li>
+          <li><a href="https://www.rocketsoftware.com/zos-open-source/tools" target="blank">IBM z/OS</a></li>
+          <li><a href="https://download.sinenomine.net/clefos/7/base/" target="blank">ClefOS</a></li>
+          <li><a href="https://packages.debian.org/stable/allpackagesx" target="blank">Debian 10 "buster"</a></li>
```

Comment:
There's a stray "x" at the end of this URL, but either way, it includes packages for all architectures, so isn't really useful. We could link to https://www.debian.org/distrib/packages and instruct the user to select the s390x architecture in the second search tool on that page. For openSUSE we can also do similar, since they also allow you to search by architecture.